src/main/java/com/zilen/weather/service/MainMenu.java

```diff
@@ -0,0 +1,29 @@
+package com.zilen.weather.service;
+
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.util.Scanner;
+
+@Component
+public class MainMenu {
+
+    private final String appid = "50b4a9bfff0df818b269d4a468128442";
+
+    @PostConstruct
+    public void inputData(){
```

Comment:
Method with @PostConstuct should make something after bean initialization, but here you try to listen console input it's a not good idea to place such code here.